documentation/staging/content/navigate/image.md

```diff
@@ -194,8 +194,8 @@ the script itself.  This change is left as an exercise for you because different
 existing standards for securely handling such credentials.
 
 ### Create Images
-The `Create Primary Image` and `Create Auxiliary Image` actions invoke the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/) to
-create a new container image for running a WebLogic domain in a Kubernetes environment.  You can access these actions using
+`Create Primary Image` and `Create Auxiliary Image` invoke the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/) to
+create a new container image for running a WebLogic domain in a Kubernetes environment.  You access these actions using
```

Comment:
technically, the "primary image" is the one used for running the domain and the auxiliary contains data that defines the domain.  That may be too much information but just wanted to make sure we were clear on the images purposes.